tests/test_clients.py

```diff
@@ -1652,6 +1652,59 @@ def test_create_document_with_content(self):
         content.seek(0)
         self.assertEqual(posted_content.read(), content.read())
 
+    def test_create_document_without_content(self):
+
+        identification = str(uuid.uuid4())
+        properties = {
+            "uuid": str(uuid.uuid4()),
+            "creatiedatum": timezone.now(),
+            "titel": "detailed summary",
+            "auteur": "test_auteur",
+            "formaat": "txt",
+            "taal": "eng",
+            "bestandsnaam": "dummy.txt",
+            "bestandsomvang": 17,
+            "link": "https://drc.utrechtproeftuin.nl/api/v1/enkelvoudiginformatieobjecten/d06f86e0-1c3a-49cf-b5cd-01c079cf8147/download",
```

Comment:
I'd make this empty, afaik this was always intended to just refer to websites where a PDF is hosted or something like that, not the actual file download URL from the API (that goes in the `inhoud` field)